UI_DSM/UI_DSM.Client/Program.cs

```diff
@@ -40,15 +43,40 @@ public static async Task Main(string[] args)
             builder.RootComponents.Add<App>("#app");
             builder.RootComponents.Add<HeadOutlet>("head::after");
 
+            AddServices(builder);
+            AddViewModels(builder);
+
+            await builder.Build().RunAsync();
+        }
+
+        /// <summary>
+        /// Register all ViewModels into the <see cref="WebAssemblyHostBuilder"/>
+        /// </summary>
+        /// <param name="builder">The <see cref="WebAssemblyHostBuilder"/></param>
+        private static void AddViewModels(WebAssemblyHostBuilder builder)
+        {
+            builder.Services.AddTransient<IUserManagementViewModel, UserManagementViewModel>();
```

Comment:
Maybe do something generic/auto register? Find all i****ViewModel interfaces and their implementations and auto register those.

Reply:
Implemented in the second commit, can you confirm that is what you though ?

Reply:
Yes, this is what I meant.